src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -172,6 +187,10 @@ export const useSwapper = () => {
 
   const { showErrorToast } = useErrorHandler()
 
+  const accountSpecifiersList = useSelector(selectAccountSpecifiers)
+
+  const { chainAdapterManager } = usePlugins()
```

Comment:
if you want to avoid using the hook and plumbing this around through effects everywhere, you can get the underlying singleton directly

https://github.com/shapeshift/web/blob/develop/src/context/PluginProvider/PluginProvider.tsx#L30